### ISSUE TITLE
Question about valid domains in manifest for fetch requests

### ISSUE THREAD

**Issue by abianche** [MicrosoftDocs/msteams-docs#2657]
Hi, in many docs I see that to display content from a URL in a Personal Tab, you need to have such URL set as valid domain in the app manifest. This is fine.

What is not clear for me is, if I have to make a fetch request from that personal Tab, for example something like this:
```jsx
 useEffect(() => {
    fetch(`https://restcountries.eu/rest/v2/name/italy`)
      .then((res) => res?.json())
      .then((json) => setCountry(json));
  }, []);
  ```

do I have to add the domain (i.e. restcountries.eu in this case) to the valid domain in the app manifest as well?
I tried this out, **without** adding the domain and it just works, but I wonder if it's because the app is not published yet - maybe there are more limitations applied afterwards.

Basically, my goal is that I need to make REST requests to a URL which is not known at compile time.
Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0f94a488-d636-92b3-21eb-c88744c8f272
* Version Independent ID: 7b3413f4-9ade-accf-5cb2-83fead11a32a
* Content: [Manifest schema reference - Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema)
* Content Source: [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/master/msteams-platform/resources/schema/manifest-schema.md)
* Product: **msteams**
* GitHub Login: @laujan
* Microsoft Alias: **lajanuar**

**Comment by RamaMohanaChary** [comment]
Hi @abianche,

Thanks for reaching us!

Could you please again test after deployment of your app, if the url does not render then we have to place it in Valid Domains of the manifest. And also we have to make sure the 'url' should match the Tab requirements.

For Tab requirements, please have a look on this [document](https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/tab-requirements).

**Comment by abianche** [comment]
Hi @RamaMohanaChary,

I am not sure if I made my question clear, but I am not trying to "render" something from such URL, I am just making an HTTP requests within the Tab code.

Can I make HTTP requests to ANY domains within the Tab? After packaging the application and sideload it to MS Teams, it seems I can definitely do this, without having to add the HTTP request URL in the valid domains of the manfiest. Of course, the manifest contains the URL with which I "render" the Tab content, but this is different.
Thanks.

**Comment by RamaMohanaChary** [comment]
Hi @abianche,

Yes it is possible to make a http request from a tab. Few url's which does not render by teams then we should provide those url's in manifest valid domains.
